book/login/intro.md

```diff
@@ -5,7 +5,7 @@ Login plugins are used in order to integrate the following capabilities:
 * User authentication: authenticate existing users with the backend
 * User authorization: after authenticating, make sure the user has a valid entitlement for an action
 * Offering various payment models: usually through in app purchases like subscriptions and one time purchases
-* Expose user profile within the app for various uses such as analytics
+* Expose user profile within the app for various uses such as analytics or personalisation
```

Comment:
Fix typo - `personalisation ` should be `personalization`

Reply:
Change the sentence to: `Expose user profile within the app for various uses such as analytics and personalization`.

Reply:
@snooky23 - side note - this is not a type-o its the difference between "American" and "British" type spelling.
That said - we are keeping it American style in the portal so it should indeed be with a z

Reply:
:D